src/ios/unlinkAssets.js

```diff
@@ -17,10 +17,10 @@ module.exports = function unlinkAssetsIOS(files, projectConfig) {
   const assets = groupFilesByType(files);
   const plistPath = path.join(projectConfig.sourceDir, getPlistPath(project));
 
-  if (!fs.existsSync(plistPath)) {
+  if (!plistPath || !fs.existsSync(plistPath)) {
```

Comment:
Maybe this should go to the standalone file? You use it at least twice in your code